Makefile

```diff
@@ -66,6 +66,7 @@ CONFIG_IMG=$(DIST)/config.img
 
 BIOS_IMG=$(DIST)/bios/OVMF.fd
 EFI_PART=$(DIST)/bios/EFI
+CONF_PART=$(CURDIR)/../adam/run/adam/config
```

Comment:
Can we change this to `CONF_PART?=$(CURDIR)/../adam/run/adam/config`? That way I can override it by doing:

```
make run CONF_PATH=/some/other/path
```

Reply:
You don't have to change anything ;-) VARs explicitly set on the command line like that take precedence (at least in GNU make -- which is the only make that can parse our Makefile anyway).

Reply:
The funny thing is, I have never tried it by overriding when `A=b`. I _always_ set it as `A?=b` so that it says, "override me if you need". But of course you are 100% correct. Sigh.